mysql/makefile

```diff
@@ -0,0 +1,19 @@
+build:
+	docker build -t mysql-volumeless:8.0 .
+
+run:
+	docker run --name mysql-volumeless \
+		-e MYSQL_USER=moov \
+		-e MYSQL_PASSWORD=secret \
+		-e MYSQL_ROOT_PASSWORD=secret \
+	       	mysql-volumeless:8.0
+
+stop:
+	docker stop mysql-volumeless
+
+image:
+	docker commit mysql-volumeless mysql-volumeless:8.0
```

Comment:
Could you describe the differences between a volumeless mysql image and regular mysql image?

Since we're only launching one container, how much is the performance gain?

Reply:
Original container stores all databases inside volume. It keeps data between container restarts.

MySQL requires initial (system) dB to be created and setup as well as user’s database. It all happens inside volume.

Custom image is a snapshot of initialized databases but instead of volume it stores them inside container.

Container disk space is limited by Docker and by default is 10Gb.

Reply:
Here is the Dockerfile: https://github.com/moov-io/base/blob/2b9526b91772e2ad5fa0c2ae868188747fe440e1/mysql/Dockerfile

Reply:
Also, in Paygate we run tests in parallel. At some pint I saw 10+ MySQL containers running... I've not measured consumed memory, but with one mysql container it will be 10 times less :D